synapse/lib/storm.py

```diff
@@ -921,6 +971,15 @@ def _stormOperLift(self, query, oper):
 
         [query.add(tufo) for tufo in self.stormTufosBy(by, prop, valu, limit=limit)]
 
+    def _stormOperLimit(self, query, oper):
+        args = oper[1].get('args',())
+        if len(args) != 1:
+            raise s_common.BadSyntaxError(mesg='limit(<size>)')
+
+        size = int(args[0])
+        if query.size() > size:
+            [ query.add(node) for node in query.take()[:size] ]
```

Comment:
This feels like a seatbeat step in the event that the previous operator did not heed the limit kwlist valu.  Is that correct?